src/main/java/uk/gov/justice/digital/delius/service/OffenderIdentifierService.java

```diff
@@ -16,14 +16,17 @@
     private final boolean updateNomsNumberFeatureSwitch;
     private final OffenderTransformer offenderTransformer;
     private final OffenderRepository offenderRepository;
+    private final SpgNotificationService spgNotificationService;
 
     public OffenderIdentifierService(
             @Value("${features.noms.update.noms.number}") Boolean updateNomsNumberFeatureSwitch,
```

Comment:
An observation but would it pay to be clear with defaulting this to 'false'. `${features.noms.update.noms.number:false}`

And why object and not primitive? meh.

Reply:
Yeah, should be little `b` - not sure about default since it is default in application.yaml anyway and further defaulted in terraform. So it would probably just be a 3 level of defaults.